pkg/driver/centos.go

```diff
@@ -10,8 +11,15 @@ type CentOSDriver struct {
 	systemdUnitOverwriteDriver
 }
 
-func (c CentOSDriver) InstallDocker() error {
-	return executil.ExecPipe("/bin/sh", "-c", "curl -sSL https://get.docker.com/ | sh")
+func (c CentOSDriver) InstallDocker(azureEnv string) error {
+	switch azureEnv {
+		case "AzureCloud":
+			return executil.ExecPipe("/bin/sh", "-c", "curl -sSL https://get.docker.com/ | sh")
```

Comment:
ohh I didn't even realize you're updating the centos driver and ubuntu driver separately. we shouldn't be doing that.

I am not sure why we are using curl vs wget in different drivers. If you could verify one of them exists in both, we can use that one in both. Converging to curl would be nice. Thanks for noticing.